components/Loading.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
```

Comment:
this import isn't used, is it? Running `yarn pretty` should also clean up any other unused imports